### PR TITLE
Ruff and project standardization

### PR DESCRIPTION
**Pull request by bandophahita**
implementing ruff and project standardization
https://github.com/ScreenPyHQ/screenpy/issues/127

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 ===============
+
+[![Build Status](../../actions/workflows/tests.yml/badge.svg)](../../actions/workflows/tests.yml)
+[![Build Status](../../actions/workflows/lint.yml/badge.svg)](../../actions/workflows/lint.yml)
+
+[![Supported Versions](https://img.shields.io/pypi/pyversions/screenpy_appium.svg)](https://pypi.org/project/screenpy_appium)
+[![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
```
Comment (perrygoy) [inline README.md:11]: Yay, i always forget about these.

File: README.md [hunk 2]
```diff
@@ -49,16 +59,23 @@ or
 
 
 Documentation
-----------
+-------------
```
Comment (perrygoy) [inline README.md:62]: Oh nice catch here 👍

File: README.md [hunk 3]
```diff
@@ -49,16 +59,23 @@ or
...
 Please check out the [Read The Docs documentation](https://screenpy-appium-docs.readthedocs.io/en/latest/) for the latest information about this module!
 
+You can also read the [ScreenPy Docs](https://screenpy-docs.readthedocs.io/en/latest/) for more information about ScreenPy in general.
```
Comment (perrygoy) [inline README.md:65]: I think this is a good idea—i do link to the ScreenPy docs on the first page of all of the extension docs, but maybe they want to start there from here.

File: screenpy_appium/__version__.py [hunk 4]
```diff
@@ -1,29 +1,34 @@
...
+                .s5SSSs.  .s5SSSs.  .s5SSSs.  s.  .s    s.  .s5ssSs.
+                      SS.       SS.       SS. SS.       SS.    SS SS.
+                sS    S%S sS    S%S sS    S%S S%S sS    S%S sS SS S%S
+                SS    S%S SS    S%S SS    S%S S%S SS    S%S SS :; S%S
+                SSSs. S%S SS .sS::' SS .sS::' S%S SS    S%S SS    S%S
+                SS    S%S SS        SS        S%S SS    S%S SS    S%S
+                SS    `:; SS        SS        `:; SS    `:; SS    `:;
+                SS    ;,. SS        SS        ;,. SS    ;,. SS    ;,.
+                :;    ;:' `:        `:        ;:' `:;;;;;:' :;    ;:'
+"""
```
Comment (perrygoy) [inline screenpy_appium/__version__.py:21]: Beautiful 👏 👏 👏

File: setup.py
Comment (perrygoy) [inline setup.py:1]: Weren't we removing `setup.py` during standardization? I may not have read your response in another PR.

File: tests/test_copyright.py
Comment (perrygoy) [inline tests/test_copyright.py:1]: I'm so glad you thought of these tests, great idea.

**Review by perrygoy — APPROVED** [review]
Standards achieved—thanks, @bandophahita! 

I had one question about `setup.py`, but it's not blocking.

File: README.md (hunk 2 above)
Comment (bandophahita) [reply README.md:62]: just wait till you see how I found it.

File: README.md (hunk 3 above)
Comment (bandophahita) [reply README.md:65]: hehe.. something something.. cruft..

File: screenpy_appium/__version__.py (hunk 4 above)
Comment (bandophahita) [reply screenpy_appium/__version__.py:21]: took me 20 minutes to find the ascii generator for this.  *sigh*

File: setup.py
Comment (bandophahita) [reply setup.py:1]: I figured that would be it's own set of PRs.  Just to be sure.

**Pull request merged**